shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -213,6 +253,11 @@ private void configureWindowForTransparency() {
     }
   }
 
+  @NonNull
+  private View createFlutterView() {
+    return delegate.onCreateView(null, null, null);
```

Comment:
nit: can you turn them into self-documenting static consts (that = null) so people reading the code can know what this is supposed to be at a glance?

Reply:
I'm not sure what you mean. Can you elaborate?

Reply:
Ah, ok, I think I'd much rather go with the comments and avoid adding new code solely for documentation. BTW, every Android developer will be very familiar with this method.

Reply:
Added comments.